docs/changing-ownership/change-ownership-or-username.md

```diff
@@ -0,0 +1,18 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Change Instance Ownership or Username
+Description: Change instance ownership or username.
+contentType: howto
+---
+
+# Change instance ownership
+
+You can change the ownership of an instance by navigating into the **Settings** page in the owner's account and editing the **Email** field. After making the changes, scroll down and press **Save**.
+Note that for the change to be effective, the new email address cannot be linked to any other n8n account.
+
+Changing emails will change the owner of the instance, the email you log in with, and the email your invoices and general communication gets sent to.
+
+# Change instance username
+
+It is not currently possible to change usernames.
+If you want your instance to have a different name you need to create a new account and transfer your work into it.
```

Comment:
Maybe explain how they can transfer work into the new account by linking this page: https://docs.n8n.io/workflows/export-import/ ?

Reply:
great point!